src/journal/test/JournalTest.hs

```diff
@@ -742,7 +757,7 @@ genHistory pids initModel = sized $ \s ->
           go (consH (Ok pid resp) conc) linear model size ((pid, DoingNothing):pids')
 
 prop_lineariseIsOkay :: Property
-prop_lineariseIsOkay = mapSize (max 20) $
+prop_lineariseIsOkay = mapSize (min 20) $
```

Comment:
I don't think you want `min 20` here it might just take more time than needed.

Reply:
I think `max 20` was wrong if we want same semantics as `withMaxSize 20`? `min 20` will allow smaller sizes but cap the size at 20.

Reply:
Oh you are right